### DISCUSSION TITLE
github-composite-actions

### DISCUSSION THREAD

**Discussion by giscus** [colindembovsky/colindembovsky.github.io#1] [Giscussions]
# github-composite-actions

Composite Actions now allow you to run other Actions, not just script steps. This is great for composability and maintainability, but there are some limitations that you should be aware of.

http://localhost:4000/github-composite-actions

**Comment by colindembovsky** [comment]
This is a cool post!

**Reply by sdolgin** [reply]
I like the blog setup here. Nice work Colin.

**Comment by kmadof** [comment]
Really nice article. I like the most part where you show limitation of composite actions. This is so important as many people is afraid of what could happen to Azure Pipelines and without reaseach they can decide to migrate to early. Of course all depends what you use in Azure Pipelines, but as you show in some cases it could be still quite painfull. Still, I'm pretty sure it will change in the future.

**Comment by wmariuss** [comment]
Does this work with a private Org and using a common repo?
